src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -63,6 +63,54 @@ protected void initializeDrivingVariants() {
         //Note: the intervals for the driving variants are set in onStartup
     }
 
+    /**
+     * A method to allow a user to inject data sources after initialization that were not specified as command-line
+     * arguments.
+     * @return The {@link FeatureInput} used as the key for this data source.
+     */
+    @SuppressWarnings({"unchecked", "rawtypes"})
+    protected FeatureInput<? extends Feature> injectFeatureDataSourcesAfterInitialization(final String filePath,
+                                                               final String name,
+                                                               final Class<? extends Feature> featureType) {
+        //TODO: Find a better way to do this.
+
+        final FeatureInput<? extends Feature> featureInput = new FeatureDataSourceWrapperClass(featureType).createFeatureInput(filePath, name);
```

Comment:
As discussed in person, I think you can do what you want here without the need for `FeatureDataSourceWrapperClass` at all.

Reply:
Yup - fixed!